features/admin.validation.v1/pages/validation-config-edit.tsx

```diff
@@ -600,14 +603,21 @@ export const ValidationConfigEditPage: FunctionComponent<MyAccountSettingsEditPa
     ): void => {
         if (hasPasswordExpiryRuleErrors) return;
 
-        const processedFormValues: ValidationFormInterface = {
+        let processedFormValues: ValidationFormInterface = {
```

Comment:
We can keep this const and add new object properties.

const obj = {
  a: "b",
  c: "d"
}

obj.e = "f"